### ISSUE TITLE
[TRAIN]  "🧠 Feat - Train/Registry" # 2

### ISSUE THREAD

**Issue by TeethChatterDev** [AIBootcamp16/mlops-cloud-project-mlops-7#3]
### 목표(모델/데이터/지표)

description: 모델 레지스트리/별칭/학습 스모크
title: "[TRAIN] "
labels: [feature, training]
body:
  - type: textarea
    id: goal
    attributes:
      label: 목표
      value: |
        - 모델 별칭(MODEL_ALIASES) → 정식명 매핑
        - get_model(task,name,params)로 회귀/분류 모델 생성
        - get_available_models(task) 제공
        - 최소 학습/예측 스모크 통과
    validations: { required: true }

  - type: textarea
    id: aliases
    attributes:
      label: 별칭 정의(현재 코드)
      value: |
        randomforest→rf, random_forest→rf, lightgbm→lgbm, xgboost→xgb, catboost→cat, logistic→logistic_regression

### 작업

  - [x] MODEL_ALIASES: randomforest→rf, random_forest→rf, lightgbm→lgbm, xgboost→xgb, catboost→cat, logistic→logistic_regression
  - [x]  get_model 회귀/분류 분기 + 미설치 가드 문구 확인
  - [x]  get_available_models 정상 반환
  - [x] 스모크: 작은 데이터로 fit/predict OK 
       

### 완료기준(AC)

        - 별칭/정식명 모두 생성됨
        - 회귀/분류 최소 1종 fit/predict 성공
        - 미설치 시 안내 메시지 표출
